app/models/task.rb

```diff
@@ -269,16 +269,54 @@ def trigger_start
     return 0 if self.root?
 
     case
-    when on_started?
-      return self.parent.trigger_start
-    when on_done?
-      return (self.parent.trigger_start + self.parent.duration)
-    else
-      return "case not coded yet"
+      when on_started?
```

Comment:
I believe the Ruby style for this actually does not have the 'when' clauses indented from the 'case' statement. I think it looks worse that way, but it is ruby convention everywhere I've looked.